controllers/apps/operations/ops_manager.go

```diff
@@ -57,14 +57,20 @@ func (opsMgr *OpsManager) Do(reqCtx intctrlutil.RequestCtx, cli client.Client, o
 		return nil, patchOpsHandlerNotSupported(reqCtx.Ctx, cli, opsRes)
 	}
 	// validate OpsRequest.spec
-	if err = opsRequest.Validate(reqCtx.Ctx, cli, opsRes.Cluster, true); err != nil {
+	isCreateCluster := false
+	if opsRequest.Spec.Type == appsv1alpha1.RestoreType {
+		isCreateCluster = true
+	}
```

Comment:
better to put `isCreateCluster` in opsBehaviour

Reply:
I put `isCreateCluster` in opsRequest, becuase there are serveral function using it.